packages/cli/templates/webcomponents/TypeScriptFileUpdate.ts

```diff
@@ -1,4 +1,4 @@
-import { App, FS_TOKEN, IFileSystem, TypeScriptUtils, Util } from "@igniteui/cli-core";
```

Comment:
in the end entire `TypeScriptFileUpdate` is mostly the same class exported in `cli.core`. I believe this was added here when POC was created. It is now time to remove this class and use the one from `core`. The changes added in this class should be transferred to the one in the `core`.

Reply:
There is an issue about that - https://github.com/IgniteUI/igniteui-cli/issues/983